WMFComponents/Sources/WMFComponents/Components/Activity Tab/WMFActivityTabViewModel.swift

```diff
@@ -0,0 +1,62 @@
+import Foundation
+import SwiftUI
+import WMFData
+
+@objc public class WMFActivityTabViewModel: NSObject, ObservableObject {
```

Comment:
Does this need to be `@objc` / `NSObject`? It doesn't look like you call the view model from Objective-C. I think you can delete those.